danish/the_book/the_book.md

```diff
@@ -0,0 +1,7 @@
+Du opdager at bogen ikke er til at finde!
+Naasaaren Benny har stjålet bogen. 
+Du opdager en kartoffel ligge hvor bogen lå og mistænker kartoffelavler Sand for at værre i besiddelse af bogen.
+
+Hvad vælger du ?
+[Du begiver udspionerer kartoffelavlerens hus?](https://www.youtube.com/watch?v=0D1Cnf5OFdc)
+[Giver op og må indse at bogen er tabt?](https://www.youtube.com/watch?v=YO4yhBVjnDY)
```

Comment:
There is a link on this line and also on the previous line.  Please make sure to insert a blank line between links.  That will help keep options visually separated.